Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -206,6 +215,15 @@ public void insertConcept(ConceptEntry entry, String userName) throws LuceneExce
                         doc.add(new StringField(searchFieldAnnotation.lucenefieldName() + LuceneFieldNames.UNTOKENIZED_SUFFIX,
                                 String.valueOf(contentOfField), Field.Store.YES));
                     }
+
+                    if (searchFieldAnnotation.isSortAllowed()) {
+                        String value = StringEscapeUtils
+                                .unescapeHtml4(String.valueOf(contentOfField).replaceAll("\\s", "")).trim()
+                                .toLowerCase();
```

Comment:
Couple of things here:
- it seems like this line is used more often; make that a method to call
- why do you remove whitespaces? 'hello world' is very different 'helloworld' and should be sorted to different positions
- html contents should be removed for sorting (the users only sees 'pony' and '**pony**' and would see why they would be on different positions.
- non-ascii characters like umlauts, accents, etc. should probably be turned into ascii for sorting (ö -> o, é -> e, etc.)